runtime/jvmti/jvmtiStartup.c

```diff
@@ -23,6 +23,12 @@
 #include <stdlib.h>
 #include <stdio.h>
 #include <string.h>
+#ifdef WIN32
+/* windows.h defined UDATA.  Ignore its definition */
+#define UDATA UDATA_win32_
+#include <windows.h>
+#undef UDATA	/* this is safe because our UDATA is a typedef, not a macro */
+#endif /* WIN32 */
```

Comment:
If this is added to get `CP_ACP` defined, I'd rather we had a J9/OMR version of this constant that we can reuse rather than doing this in our files

Reply:
Yeah, am going to use J9/OMR API instead.